src/adapter/ads/nativeads.js

```diff
@@ -49,30 +50,101 @@ let NativeAdsAdapter = youbora.StandardAdapter.extend({
     return returnValue;
   },
 
-  /**  @returns {void} - Return a list of events and methods to suscribe from this.player. */
-  getListenersList: function() {
+  /** @returns {object} - given ad structure (list with number of pre, mid, and post breaks) */
+  getGivenBreaks: function() {
+    return this.adBreaks ? this.adBreaks.length : null;
```

Comment:
@jesuslc91 this will give you only the number of ad breaks, not a list as described in comment.

Reply:
You are right, the text is wrong. Updating!